### DISCUSSION TITLE
How to copy commit's SHA?

### DISCUSSION THREAD

**Discussion by skoch13** [NeogitOrg/neogit#1247] [Q&A] [answered]
I want to quickly copy commit's SHA, but pressing 'y' opens hidden gem - NeogitRefsView. Am I doing it wrong?

**Comment by CKolkey** [comment, marked as answer]
`Y` will do that :) Just about everything in the interface responds to `Y` with something. A commit hash, a file path, etc.

**Reply by skoch13** [reply]
As always you're awesome! Thank you

**Reply by skoch13** [reply]
Maybe it's worth mentioning this anywhere within the documentation?

**Reply by CKolkey** [reply]
hmm.. Yeah, I could add more. Currently there's the mapping config for "YankSelected", but that without any description of what it does.

**Reply by tienlonghungson** [reply]
> `Y` will do that :) Just about everything in the interface responds to `Y` with something. A commit hash, a file path, etc.

Hi, that is amazing. What about copy commit's url? I don't know how to do that?

**Reply by CKolkey** [reply]
`o` from the commit view opens it in your browser

**Reply by tienlonghungson** [reply]
I think that is great. But can it be more convenient? For example, in lazygit, under the selected commit, pressing `y`, there is a list of options that we can choose to copy (see the picture)
<img width="1479" height="341" alt="image" src="https://github.com/user-attachments/assets/21f827cf-be66-4ba9-8b71-dcc51450ed80" />
